core/background.go

```diff
@@ -23,7 +23,7 @@ var backgroundTaskStartDelay = flags.EnvInt("BACKGROUND_TASK_START_DELAY", 0)
 func backgroundTaskLoop() {
 	datadog.Info("Waiting %d seconds to start background jobs.", backgroundTaskStartDelay)
 	time.Sleep(time.Second * time.Duration(backgroundTaskStartDelay))
-	datadog.Info("Starting background jobs.")
+	datadog.Info("Starting background jobs. %s", "")
```

Comment:
So, you should be able to do `datadog.Info("Starting background jobs.")` without it added extra text. I looked into it a bit and it seems like the issue is that there's a few places where you pass through `args` directly, but instead you should do `args...`, which fixes it. By passing in `args`, you're passing it in as a slice directly, rather than variadic parameters. So if you check for the places in `datadog.go` and `logger.go` where you do `[...] format, args)`, you should be able to change them to `[...] format, args...)` and fix this.